### PR TITLE
Increase getBlockInfo coverage to 100%

### PR DESCRIPTION
**Pull request by AmberKiso**
## Summary
Increase code coverage of getBlockInfo

## Testing Plan
yarn test:coverage:html
yarn lint

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

**Comment by AmberKiso** [comment]
Thinking I remove the troublesome case - it's failing intermittently.

File: ironfish/src/indexers/__fixtures__/minedBlocksIndexer.test.ts.fixture [hunk 1]
```diff
@@ -480,6 +480,82 @@
           "data": "base64:AAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAAAAAKvNGf/TZnmC55LsNvg7V2ak+sIUYAeGzWHLfw/hD41GnrQcSwyJs4N/woPMNmwZ0pZpBT8GtW2BT2s+Q9x4bCM8pOvlTDX2a5RvFyewHgGbw4cr6/bWJpqr27rMv/m4rxm4ExR3tzsDpTofI0iG16APULIz3XtaixNVndvV+37QKCpz31+OyGGu4Z7Wn2lvN7kTczomJcrKRmmBibOSJSW/hpCW41H+L5aZmX1Igc4UqUTxEQLQrLQ07GYsx4G/SRb2vYT6OBnZB0A1a7ya2M0OOdXUZe4jJgJal2SbpzasbdiqWpaCzCcg/0UYMLGvRy87Rd5Exwa7Dxng3mszkiIkjz72IicdaAX1scxUIXmDCL657CeEvAehfV7hHv/paa914wo7L24lFGFAfcefTJPe7Z7DnOrnpdZqAMYKBU9AlQjJLLt/xP2sSe9hn51FBRzHoDZQUZM7ewQB2PyZtzOikdknzGyLjcqYImMzKXLu7jf/iSS65JakYEwyFE/selD+7EJlYW5zdGFsayBub3RlIGVuY3J5cHRpb24gbWluZXIga2V5MDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwYtJSoFPdK+ptLxbDokNkVQoRMjzGLJyjCaq5NJCm52hMzG812shYowmk7NLtCwPfehjXY/9c01BD55lMlsjPCg=="
         }
       ]
+    },
+    {
+      "id": "a629648c-d7d9-49c1-a751-3837391ed0e4",
```
Comment (dguenther) [inline ironfish/src/indexers/__fixtures__/minedBlocksIndexer.test.ts.fixture:485]: The changes in this file can be reverted -- the minedBlocksIndexer test wasn't changed in this PR.

File: ironfish/src/rpc/routes/chain/getBlockInfo.test.ts [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      if (!(e instanceof RpcRequestError)) {
+        throw e
+      }
+      expect(e.status).toBe(400)
+      expect(e.code).toBe(ERROR_CODES.VALIDATION)
+      expect(e.message).toContain('No block with header')
+    }
+  })
+
+  it('Receives transactions from a matched block', async () => {
```
Comment (dguenther) [inline ironfish/src/rpc/routes/chain/getBlockInfo.test.ts:118]: I think you're missing an additional fixture for this test that should make it run faster. When I ran the tests locally, git showed additional changes to `getBlockInfo.test.ts.fixture`.

**Review by dguenther — CHANGES_REQUESTED** [review]

File: ironfish/src/rpc/routes/chain/getBlockInfo.test.ts [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    //Create separate test case for showing transactions
+    const { chain, node } = routeTest
+
+    await chain.open()
```
[the remark this answers is not in the document]
Comment (AmberKiso) [reply ironfish/src/rpc/routes/chain/getBlockInfo.test.ts:122]: Will do

File: ironfish/src/indexers/__fixtures__/minedBlocksIndexer.test.ts.fixture (hunk 1 above)
Comment (AmberKiso) [reply ironfish/src/indexers/__fixtures__/minedBlocksIndexer.test.ts.fixture]: Oops. That's been removed.

File: ironfish/src/rpc/routes/chain/getBlockInfo.test.ts (hunk 2 above)
Comment (AmberKiso) [reply ironfish/src/rpc/routes/chain/getBlockInfo.test.ts:118]: Done

**Pull request merged**